core-java/src/test/java/com/baeldung/utils/UtilsTest.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.utils;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UtilsTest {
```

Comment:
UtilsTest is a little bit too generic name. I can imagine multiple classes with similar names after having published hundreds of articles :)